source/develop/release-management/features/storage/backup-storage-domain.html.md

```diff
@@ -0,0 +1,248 @@
+---
+title: BackupStorageDomain
+category: feature
+authors: mlipchuk, shubham0d
+wiki_category: Feature|BackupStorageDomain
+wiki_title: Features/BackupStorageDomain
+wiki_revision_count: 0
+wiki_last_updated: 2017-24-03
+feature_name: Backup Storage Domain
+feature_modules: engine/vdsm
+feature_status: Yet to Impelement
+--- 
+
+
+# Backup Storage Domain
+
+This feature will add another functionality layer to the oVirt Disaster Recovery solution.
```

Comment:
Let's beef this up a bit - what's the problem you're trying to solve with this feature? How about a sentence explaining that it's another step in deprecating the export domain?